src/components/application_manager/src/mobile_message_handler.cc

```diff
@@ -104,7 +104,7 @@ application_manager::Message* MobileMessageHandler::HandleIncomingMessageProtoco
   }
   LOG4CXX_DEBUG(logger_, "Incoming RPC_INFO: " <<
                          (out_message->connection_key() >> 16) <<", "<<
-                         GetMessageTypes[out_message->type()] <<", "<<
+                         //GetMessageTypes[out_message->type()] <<", "<<
```

Comment:
I'm not sure, @AGaliuzov, that I will remember after a week of deep-asap implementation that I need to  return some deleted code without any remarks (todo-s or jira tickets).
An also, commented code is not get a clue what was wrong: unimplemented stream, crashing, dead-lock, pasa-specific build fail?. But comment can do.